services/movie_session.py

```diff
@@ -0,0 +1,48 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(
+        movie_show_time: datetime.date,
```

Comment:
The `movie_show_time` parameter is typed as `datetime.date`, but it should be `datetime` to match the `DateTimeField` type of `show_time` in the `MovieSession` model.